src/CommunityToolkit.Maui.Core/Essentials/FileFolderDialog.tizen.cs

```diff
@@ -0,0 +1,398 @@
+﻿using Microsoft.Maui.Platform;
+using Tizen.NUI;
+using Tizen.NUI.BaseComponents;
+using Tizen.UIExtensions.NUI;
+using Path = System.IO.Path;
+using Button = Tizen.UIExtensions.NUI.Button;
+using Entry = Tizen.UIExtensions.NUI.Entry;
+using Label = Tizen.UIExtensions.NUI.Label;
+using TColor = Tizen.UIExtensions.Common.Color;
+using ScrollView = Tizen.UIExtensions.NUI.ScrollView;
+using Shadow = Tizen.NUI.Shadow;
+using HorizontalAlignment = Tizen.NUI.HorizontalAlignment;
+using VerticalAlignment = Tizen.NUI.VerticalAlignment;
+using View = Tizen.NUI.BaseComponents.View;
+using Window = Tizen.NUI.Window;
+
+namespace CommunityToolkit.Maui.Storage;
+
+sealed class FileFolderDialog : Popup<string>
```

Comment:
Hey @JoonghyunCho! Could you confirm that `FileFolderDialog.tizen.cs` only contains Tizen UI Code and doesn't contain any .NET MAUI UI code. I.e., It doesn't contain any code  from the `Microsoft.Maui.Controls` namespace like `Microsoft.Maui.Controls.Button`.

I think we're good, but I just want to double-check that we're not accidentally putting .NET MAUI UI code in `CommunityToolit.Maui.Core`: 
https://github.com/CommunityToolkit/Maui/blob/main/CONTRIBUTING.md#where-to-make-your-changes

[![](https://user-images.githubusercontent.com/13558917/145694198-7addbd35-0e5f-4816-b351-759a01ec2672.png)](https://github.com/CommunityToolkit/Maui/blob/main/CONTRIBUTING.md#where-to-make-your-changes)

Reply:
Thanks for the double check! No Maui code is included in the `FileFolderDialog.tizen.cs`. It only contains Tizen UI code (Tizen.NUI).